lib/bookingsync/api/client.rb

```diff
@@ -209,7 +209,8 @@ def paginate(path, options = {}, &block)
           response = call(request_method, path, options)
         end
         @pagination_first_response = response
-        data = response.resources.dup
+        data = []
+        data.concat(response.resources)
```

Comment:
this is for just small content and almost 3x faster, duping the response hash is unnecessary overhead